docs/intro/install.txt

```diff
@@ -73,8 +71,8 @@ Then at the Python prompt, try to import Django:
 .. parsed-literal::
 
     >>> import django
-    >>> print(django.get_version())
-    |version|
+    >>> django.get_version()
+    '|version|'
```

Comment:
I'm not sure I see a major benefit to this particular change in the PR. Can you help me understand why this is necessary?

Reply:
These steps are usually for new developers so, since `print` is not necessary to get the version number, I remove it. It seems to me more concise, and easier to understand.

Reply:
Thank you for the explanation! 

I think easier to understand is dependent on whether they understand that the shell will display the repr of the statement. I'm not sure that's a given for new python developers.

Reply:
Sorry I should have replied. Thank you for the explanation. I wouldn't include this change, but since nobody else has said anything, it may be fine. I'll trust your judgement.